lib/fiet/feed.ex

```diff
@@ -127,6 +127,7 @@ defmodule Fiet.Feed do
   defp text_construct({_type, content}), do: content
   defp text_construct(nil), do: nil
 
+  defp extract_atom_link(links) when is_list(links), do: extract_atom_link(List.last(links))
```

Comment:
It's quite hard to determine which link to use when there are multiple links. Could you please update the related documentation to clarify our choice?

Also I think choosing the first link instead would be more efficient, if there is no clear benefit of it being the last one.

Reply:
You are absolutely correct! So I changed the logic a bit more. The correct extracted link should be the one with the `rel` attribute set to `self`. If that one does not exist for the given feed, do a fallback on the first link found in the list.